src/components/Entities/Entity.scss

```diff
@@ -5,69 +5,91 @@
   font: #{$cellSize}/#{$cellSize} "Apple Color Emoji";
   z-index: 1;
 
-  &--ambulance::before  { content: "🚑"; }
-  &--banana::before     { content: "🍌"; }
-  &--bee::before        { content: "🐝"; }
-  &--bike::before       { content: "🚴"; }
-  &--boots::before      { content: "👢"; }
-  &--building::before   { content: "🏢"; }
-  &--cactus::before     { content: "🌵"; }
-  &--camel::before      { content: "🐫"; }
-  &--carA::before       { content: "🚗"; }
-  &--carB::before       { content: "🚙"; }
-  &--chicken::before    { content: "🐣"; }
-  &--chopper::before    { content: "🚁"; }
-  &--cloud::before      { content: "☁"; }
-  &--corn::before       { content: "🌽"; }
-  &--creepsun::before   { content: "🌞"; }
-  &--dancer::before     { content: "💃"; }
-  &--door::before       { content: "🚪"; }
-  &--elephant::before   { content: "🐘"; }
-  &--fire::before       { content: "🔥"; }
-  &--firetruck::before  { content: "🚒"; }
-  &--fishA::before      { content: "🐟"; }
-  &--fishB::before      { content: "🐠"; }
-  &--flowerA::before    { content: "🌻"; }
-  &--flowerB::before    { content: "🌹"; }
-  &--flowerC::before    { content: "🌷"; }
-  &--gator::before      { content: "🐊"; }
-  &--ghost::before      { content: "👻"; }
-  &--hammer::before     { content: "🔨"; }
-  &--heart::before      { content: "❤️"; }
-  &--houseA::before     { content: "🏠"; }
-  &--houseB::before     { content: "🏡"; }
-  &--invisible::before  { content: "  "; }
-  &--leavesA::before    { content: "🍃"; }
-  &--leavesB::before    { content: "🍂"; }
-  &--leavesC::before    { content: "🌿"; }
-  &--mart::before       { content: "🏪"; }
-  &--monkey::before     { content: "🐒"; }
-  &--musichall::before  { content: "🏫"; }
-  &--palm::before       { content: "🌴"; }
-  &--person::before     { content: "🏃"; }
-  &--personWalk::before { content: "🚶"; }
-  &--police::before     { content: "🚓"; }
-  &--rabbit::before     { content: "🐇"; }
-  &--santa::before      { content: "🎅"; }
-  &--shell::before      { content: "🐚"; }
-  &--shit::before       { content: "💩"; }
-  &--silverware::before { content: "🍴"; }
-  &--snake::before      { content: "🐍"; }
-  &--snowflake::before  { content: "❄"; }
-  &--snowman::before    { content: "⛄️"; }
-  &--speedboat::before  { content: "🚤"; }
-  &--moai::before       { content: "🗿"; }
-  &--sun::before        { content: "☀"; }
-  &--sunglasses::before { content: "👓"; }
-  &--tape::before       { content: "📼"; }
-  &--taxi::before       { content: "🚕"; }
-  &--tornado::before    { content: "💨"; }
-  &--treeA::before      { content: "🌲"; }
-  &--treeB::before      { content: "🎄"; }
-  &--treeC::before      { content: "🌳"; }
-  &--turtle::before     { content: "🐢"; }
-  &--wave::before       { content: "🌊"; }
-  &--willows::before    { content: "🌾"; }
+  .twemoji &::before, &::after {
```

Comment:
heh, no, that `::after` is leftover from an experiment of consolidating entities/ground into one element. Then I realized ground is just background colors ...